classifier/main.py

```diff
@@ -95,7 +106,8 @@ def index():
 
     # classify
     snn_dict = _classify_with_snn(alert_dict)
-    errors = gcp_utils.insert_rows_bigquery(bq_table, [snn_dict])
+    errors = bq_client.insert_rows(bq_table_id, [snn_dict], bq_schema)
+
     if len(errors) > 0:
         logger.log_text(f"BigQuery insert error: {errors}", severity="WARNING")
```

Comment:
Consider moving the BigQuery load down so that it happens after the Pub/Sub publish. That way the Pub/Sub message will still get sent even if the BigQuery load encounters problems.